chrome/content/zotero/xpcom/attachments.js

```diff
@@ -688,32 +688,13 @@ Zotero.Attachments = new function(){
 		
 		return attachmentItem;
 	});
-	
-	
+
+
+	/**
+	 * @deprecated Use Zotero.Utilities.cleanURL instead
+	 */
 	this.cleanAttachmentURI = function (uri, tryHttp) {
-		uri = uri.trim();
-		if (!uri) return false;
-		
-		var ios = Components.classes["@mozilla.org/network/io-service;1"]
-			.getService(Components.interfaces.nsIIOService);
-		try {
-			return ios.newURI(uri, null, null).spec // Valid URI if succeeds
-		} catch (e) {
-			if (e instanceof Components.Exception
-				&& e.result == Components.results.NS_ERROR_MALFORMED_URI
-			) {
-				if (tryHttp && /\w\.\w/.test(uri)) {
-					// Assume it's a URL missing "http://" part
-					try {
-						return ios.newURI('http://' + uri, null, null).spec;
-					} catch (e) {}
-				}
-				
-				Zotero.debug('cleanAttachmentURI: Invalid URI: ' + uri, 2);
-				return false;
-			}
-			throw e;
-		}
+		return Zotero.Utilities.cleanURL(uri, tryHttp);
```

Comment:
This should have a debug line:

"Zotero.Attachments.cleanAttachmentURI() is deprecated -- update your code"

(We can replace the existing calls in short order, but we'll need a reminder.)